subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_pn.c

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <errno.h>
+#include <string.h>
+#include <zephyr/sys/atomic.h>
+#include <zephyr/settings/settings.h>
+
+#include "fp_storage_pn.h"
+
+#define SETTINGS_PN_SUBTREE_NAME "fp_pn"
+#define SETTINGS_PN_KEY_NAME "pn"
+#define SETTINGS_NAME_SEPARATOR_STR "/"
+#define SETTINGS_PN_FULL_NAME \
+	(SETTINGS_PN_SUBTREE_NAME SETTINGS_NAME_SEPARATOR_STR SETTINGS_PN_KEY_NAME)
+
+BUILD_ASSERT(SETTINGS_NAME_SEPARATOR == '/');
+
+static char personalized_name[FP_STORAGE_PN_BUF_LEN] = CONFIG_BT_FAST_PAIR_STORAGE_EXT_PN_DEFAULT;
```

Comment:
Shouldn't this be removed? (as discussed yesterday)

Reply:
Yes, it should. I forgot about it.

Reply:
It's done now.